static/js/message_fetch.js

```diff
@@ -178,19 +178,6 @@ exports.initialize = function () {
         }
 
         server_events.home_view_loaded();
-
-        // backfill more messages after the user is idle
-        var backfill_batch_size = 1000;
-        $(document).idle({idle: 1000*10,
-                          onIdle: function () {
-                              var first_id = message_list.all.first().id;
-                              exports.load_old_messages({
-                                  anchor: first_id,
-                                  num_before: backfill_batch_size,
-                                  num_after: 0,
-                                  msg_list: home_msg_list,
-                              });
-                          }});
```

Comment:
Removing this fetch will cause a lot of problems; we use the data fetched here to popular recent topics in the left sidebar, ensure that typeahead, fading of low-traffic streams, etc. are done correctly.  

Note that this is not "when idle" in general, it's just a 1-time fetch 10s if idle time after the page loads.  We wait for the user to be idle just to make sure we don't impact performance when using the app.

I'll add a comment explaining the purpose of this in more detail.